src/components/NavBar.tsx

```diff
@@ -26,6 +26,7 @@ export const Navbar = ({ handleOpenDrawer }: NavbarProps) => {
   }, [location.pathname])
 
   const isReport = location.pathname === '/report'
+  const twoToneColor = '#4c88ef'
```

Comment:
i think can move to `src/constants/colors.ts`

Reply:
✔️ Done